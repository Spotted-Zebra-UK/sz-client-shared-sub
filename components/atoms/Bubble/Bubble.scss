.Bubble {
  border-radius: 50%;
  opacity: 0.1;
  position: absolute;
}
