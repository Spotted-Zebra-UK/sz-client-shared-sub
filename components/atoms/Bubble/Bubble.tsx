import './Bubble.scss';
import React, { FC } from 'react';

export interface IBubble {}

const Bubble: FC<IBubble> = () => {
  return <div className="Bubble" />;
};

export default Bubble;
