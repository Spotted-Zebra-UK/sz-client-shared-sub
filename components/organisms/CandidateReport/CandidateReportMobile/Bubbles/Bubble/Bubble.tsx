import './Bubble.scss';
import React, { FC } from 'react';

export interface IBubble {
  id?: number;
  width: string;
  height: string;
  backgroundColor: string;
  top?: string;
  right?: string;
  bottom?: string;
  left?: string;
}

const Bubble: FC<IBubble> = props => {
  const { width, height, backgroundColor, top, right, bottom, left } = props;
  const circleStyle = {
    width,
    height,
    backgroundColor,
    top,
    right,
    bottom,
    left,
  };

  return <div className="Bubble" style={circleStyle} />;
};

export default Bubble;
