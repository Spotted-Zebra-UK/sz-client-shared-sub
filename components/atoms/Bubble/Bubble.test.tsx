import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import React from 'react';
import Bubble from './Bubble';

describe('Bubble', () => {
  it('should render correctly by default', () => {
    const wrapper = shallow(<Bubble />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
