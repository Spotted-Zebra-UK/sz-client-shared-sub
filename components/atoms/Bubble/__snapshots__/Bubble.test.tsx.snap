// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bubble should render correctly by default 1`] = `
<div
  className="Bubble"
/>
`;
